### ISSUE TITLE
Checksum mismatch updating to eVision 0.1.10

### ISSUE THREAD

**Issue by kipcole9** [cocoa-xu/evision#111]
@cocoa-xu congrats on the big update. I'm updating from `0.1.8` but despite cleaning everything I'm seeing a checksum mismatch:
```elixir
kip@Kips-MacBook-Pro image % mix deps.compile evision                            

16:10:27.715 [info] EVISION_PREFER_PRECOMPILED: true; try to download and use the precompiled library.

16:10:27.718 [info] Requested version `0.1.10` has precompiled binaries.

16:10:27.718 [info] Current target `aarch64-apple-darwin` has precompiled binaries.

16:10:27.718 [info] Current NIF version `2.16` has precompiled binaries.

16:10:27.728 [info] Precompiled binary tarball cached at /Users/kip/Library/Caches/evision-nif_2.16-aarch64-apple-darwin-0.1.10.tar.gz, sha256=6af3996073ff8b6e67e854f22aa35844a7bdfed195352e61b3d686eabe3998a4

16:10:27.728 [error] Checksum mismatched: /Users/kip/Library/Caches/evision-nif_2.16-aarch64-apple-darwin-0.1.10.tar.gz[sha256=6af3996073ff8b6e67e854f22aa35844a7bdfed195352e61b3d686eabe3998a4], expected:[sha256=6209110fea311e0f5c14d459772b6dd549b96af69c3be400980a368f7b198f0b]

16:10:27.728 [warning] Will delete cached tarball /Users/kip/Library/Caches/evision-nif_2.16-aarch64-apple-darwin-0.1.10.tar.gz and re-download

16:10:28.890 [info] Precompiled binary tarball downloaded and saved to /Users/kip/Library/Caches/evision-nif_2.16-aarch64-apple-darwin-0.1.10.tar.gz, sha256=6af3996073ff8b6e67e854f22aa35844a7bdfed195352e61b3d686eabe3998a4

16:10:28.892 [error] Checksum mismatched: downloaded file: /Users/kip/Library/Caches/evision-nif_2.16-aarch64-apple-darwin-0.1.10.tar.gz[sha256=6af3996073ff8b6e67e854f22aa35844a7bdfed195352e61b3d686eabe3998a4], expected:[sha256=6209110fea311e0f5c14d459772b6dd549b96af69c3be400980a368f7b198f0b]
==> evision
could not compile dependency :evision, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile evision", update it with "mix deps.update evision" or clean it with "mix deps.clean evision"
** (RuntimeError) Checksum mismatched: downloaded file: /Users/kip/Library/Caches/evision-nif_2.16-aarch64-apple-darwin-0.1.10.tar.gz[sha256=6af3996073ff8b6e67e854f22aa35844a7bdfed195352e61b3d686eabe3998a4], expected:[sha256=6209110fea311e0f5c14d459772b6dd549b96af69c3be400980a368f7b198f0b]
    /Users/kip/Development/image/deps/evision/mix.exs:484: Mix.Tasks.Compile.EvisionPrecompiled.prepare/4
    (mix 1.14.0) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.0) lib/mix/tasks/compile.all.ex:92: Mix.Tasks.Compile.All.run_compiler/2
    (mix 1.14.0) lib/mix/tasks/compile.all.ex:72: Mix.Tasks.Compile.All.compile/4
    (mix 1.14.0) lib/mix/tasks/compile.all.ex:59: Mix.Tasks.Compile.All.with_logger_app/2
    (mix 1.14.0) lib/mix/tasks/compile.all.ex:33: Mix.Tasks.Compile.All.run/1
    (mix 1.14.0) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.0) lib/mix/tasks/compile.ex:134: Mix.Tasks.Compile.run/1
```
I've removed the cached tarball, `mix deps.clean evision` and repeated but still the same result.  Any suggestions?

**Comment by cocoa-xu** [comment]
Sorry, the checksum file I pushed to hex.pm seems to be wrong. I'll push a new release with the correct checksums.

The checksum was calculated when I pushed [c67c918](https://github.com/cocoa-xu/evision/commit/c67c91868eea0ee98185cdc03e42acd7be34c1b3), but then I found some issue with cache path, so I pushed a fix [1626748](https://github.com/cocoa-xu/evision/commit/1626748388fac4f8cc602512e86c1c224d1a9259). And I forgot to overwrite the checksum file.
